src/styles/codeStyles/types.ts

```diff
@@ -1,7 +1,7 @@
 import {TextStyle, ViewStyle} from 'react-native';
 
 type CodeWordWrapperStyles = ViewStyle;
-type CodeWordStyles = TextStyle;
+type CodeWordStyles = ViewStyle;
```

Comment:
@blazejkustra Otherwise there is a TS error in this [line](https://github.com/Expensify/App/blob/b75f5134a1be064ea252110382d5f99d6c70c981/src/components/InlineCodeBlock/index.native.tsx#L17)
![image](https://github.com/Expensify/App/assets/23176449/043d685c-714d-453e-a162-47add438edb5)

Reply:
@cubuspl42 More detailed clarification, let me know if it makes more sense to you:
- `CodeWordStyles` type is used for typing  `styles.codeWordStyle`
- `styles.codeWordStyle` is passed to the `WrappedText` component as a part of `wordStyles` param
- `wordStyles` param in `WrappedText` is used for `View` component [styling](https://github.com/Expensify/App/blob/main/src/components/InlineCodeBlock/WrappedText.js#L62)
- so it expects that `styles.codeWordStyle` will have `ViewStyle` type instead of `TextStyle`, which means `CodeWordStyles` type should be `ViewStyle`

Reply:
Ok, thanks. So I understand that we don't pass any text-specific styles, and we stick to the `View`-supported style properties subset